R/simulate_linreg.R

```diff
@@ -277,60 +213,41 @@ sim_sumstats <- function(RL, ngwas, beta, h2ge) {
 # The following is modified using chatgpt4 from
 # https://github.com/xueweic/fine-mcoloc/blob/b7aa9d27b0be13044224a4b2207717e74611d449/dsc/phenotype_simulation.r
 
-#' Simulate Multiple Traits from Genotype and Effect Sizes
```

Comment:
Your input include both G and B right? Why did you remove "effect size" related statements from documentation?